src/ds/d400/d400-color.cpp

```diff
@@ -118,14 +120,23 @@ namespace librealsense
         }
     }
 
+    void d400_color::register_features()
+    {
+        firmware_version fw_ver = firmware_version( get_info( RS2_CAMERA_INFO_FIRMWARE_VERSION ) );
+
+        if( fw_ver >= firmware_version( "5.10.9.0" ) )
```

Comment:
No need to specify a string for us to parse... can use the other ctor

Reply:
This was copied from the original code. I will update